docs/user/unattended/profile/software.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 2
+---
+
+# Additional software
+
+Depending on the [product you are installing](./product), Agama will select a set of repositories,
+patterns and packages to install. However, it is possible to add your own list of repositories,
+patterns and packages using the `software` section.
+
+This section supports three keys:
+
+- `patterns`: list of patterns to install (e.g., `gnome`, `kde`, `fips`, etc.). You can find a list
+  of patterns for your distribution using `zypper se --type pattern`.
+- `packages`: list of packages to install (e.g., `neovim`).
+- `extraRepositories`: list of additional repositories that will be used as installation source.
+
+```jsonnet
+{
```

Comment:
Let's wrap all this in `{software: ... }` and we will have a fully valid profile that I can paste into `agama config generate -` manually, or write a script to automatically validate all these snippets in CI.

Reply:
All the other sections are fine in that respect, with the notable exception of `storage`.